styles/listenbrainz/catppuccin.user.css

```diff
@@ -0,0 +1,713 @@
+            background-color: @surface2;
+          }
+          button:enabled:hover {
+            background-color: @surface1;
+          }
+          button:enabled:focus {
+            background-color: transparent;
+          }
+
+          .react-calendar__tile--hasActive,
+          .react-calendar__tile--active {
+            color: @text;
+            background-color: fade(@accent-color, 40%);
+            &:enabled:hover {
+              background-color: fade(@accent-color, 50%);
+            }
+          }
+        }
+
+        .react-calendar__month-view__days__day--neighboringMonth {
+          color: @subtext0;
+        }
+
+        .react-calendar__month-view__days__day--weekend {
+          color: @red;
+        }
+      }
+    }
+
+    .stats-full-width-graph {
+      line {
+        stroke: @subtext0 !important;
+      }
+
+      text {
+        fill: @text !important;
+      }
+
+      &.user-artist-map > div + div > div,
+      svg + div > div {
+        background-color: @overlay0 !important;
+        color: @text !important;
+      }
+    }
+    rect[fill="#353070"] {
+      fill: @mauve;
+    }
+    rect[fill="#eb743b"] {
+      fill: @peach;
+    }
+    .badge-info {
+      background-color: @surface1;
+    }
+
+    .modal-content {
+      background-color: @mantle;
+      button.close {
+        color: @text;
+        opacity: 1;
+        text-shadow: none;
+      }
+
+      .track-search {
+        #recording-mbid,
+        .track-search-dropdown {
+          background-color: @surface0;
+          color: @text;
+          border-color: @crust;
+        }
+      }
+
+      .modal-header,
+      .modal-footer {
+        border-color: @crust;
+      }
+    }
+
+    .explore-card {
+      background-color: @surface0;
+      filter: drop-shadow(0 4px 4px fade(@overlay0, 25%));
+      .explore-card-img {
+        border-color: @surface0;
+      }
+      .explore-card-text {
+        background-color: @surface0;
+      }
+    }
+
+    .sidebar {
+      background-color: @mantle;
+      .sidebar-header {
+        background-color: @crust;
+        color: @text;
+      }
+      .sidenav-content-grid {
+        background: transparent;
+      }
+    }
+    .rc-slider-rail {
+      background-color: @surface1;
+    }
+    .rc-slider-handle {
+      background-color: @accent-color;
+      &.rc-slider-handle-dragging {
+        background-color: @accent-color;
+        border-color: @accent-color;
+      }
+    }
+    .rc-slider-mark-text {
+      color: @subtext0;
+    }
+    .release-card-grid-title {
+      background-color: transparent;
+      &::before,
+      &::after {
+        border-color: @surface0;
+      }
+    }
+
+    #homepage-container {
+      @flipped-peach: if(
+        @lookup = latte,
+        @catppuccin[@mocha][@peach],
+        @catppuccin[@latte][@peach]
+      );
+      @flipped-mauve: if(
+        @lookup = latte,
+        @catppuccin[@mocha][@mauve],
+        @catppuccin[@latte][@mauve]
+      );
```

Comment:
Why is this needed?

Reply:
The [homepage](https://listenbrainz.org/?redirect=false) uses orange-to-purple gradients directly behind the text - using the matching versions of mauve and peach didn't give the text sufficient contrast to be legible. Flipping to the opposite scheme's version of the same colours seemed to do the trick.

I'm happy to try other approaches, of course.

Reply:
How does it look if you just use the colors as normal instead of the flipped versions?

Reply:
The flipped versions look like this:
![Screenshot 2024-06-16 at 11-58-38 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/3b2040e9-501f-4a49-a7e0-f0309d6fd91c)
![Screenshot 2024-06-16 at 11-58-08 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/884c7358-6245-498a-83a0-a8445d3619da)

And the unflipped versions look like this:
![Screenshot 2024-06-16 at 12-01-06 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/b58aaca5-5629-42fe-814c-4032d73317a7)
![Screenshot 2024-06-16 at 12-01-42 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/7dbe37e7-4dbd-48b8-8c89-58b15e7f8b29)

Reply:
How do the unflipped versions look if you invert the text color?

Reply:
The top half of the page looks fine:
![Screenshot 2024-07-01 at 11-57-50 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/2d11328f-7e7d-4659-82ff-937e730bd1b3)
![Screenshot 2024-07-01 at 11-58-16 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/e03d88cd-59e6-402e-8b1b-b26d987b1411)

But the bottom half has links that become illegible:
![Screenshot 2024-07-01 at 11-58-51 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/a9766f30-7dfa-4946-b070-e9abd45460dc)
![Screenshot 2024-07-01 at 11-58-51 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/0928f2ae-97d8-4869-8c2a-482458e7467a)

Colouring the links themselves differently would be an option?

Reply:
Yep let's try that.

Reply:
Since we want the links to still look like links, I tried using the blue and sky from the "flipped" flavour like I did with the background before, and that seems to look quite good (hovering over the second link to show how it looks on hover):
![Screenshot 2024-07-05 at 13-57-26 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/c75839d2-2c55-4a63-8d17-e6dc766a9140)
![Screenshot 2024-07-05 at 13-57-41 ListenBrainz](https://github.com/catppuccin/userstyles/assets/610876/f76ef585-1845-4877-a92f-ad98afacf4fc)

Are we okay to "flip" for just the link colours, keeping everything else the same, or is there a better approach I'm missing? 🤔

Reply:
Yes that sounds good.